rust/processor/src/worker.rs

```diff
@@ -111,7 +111,10 @@ bitflags! {
 
         // User transaction
         const SIGNATURES = 1 << 23;
-
+        // More tables
+        const CURRENT_TABLE_ITEMS = 1 << 24;
+        const BLOCK_METADATA_TRANSACTIONS = 1 << 25;
+        
         // Events
         const EVENTS = 1 << 24;
```

Comment:
nit:  24 is used  by current table items. maybe this will  likely happen again, should we assign specific range to each processor to prevent this conflict? like 20-30 -> default processor 30-40 -> fa. since it will be less than 10 tables

Reply:
oh. that's a good idea.